src/3_ESC.h

```diff
@@ -74,8 +75,15 @@
   - Connect a 100 - 470uF electrolytic cap and a 100nF tantalum or ceramic cap across pins 2 - 3
 */
 //#define RZ7886_DRIVER_MODE // An RZ7886 motor driver is used instead of a standard RC Crawler Type ESC. suitable for motors up to 370 size, for example WPL vehicles.
-const uint16_t RZ7886_FREQUENCY = 500;     // 500 Hz is recommended. It is not audible, if virtual engine sound is running. Higher frequencies may overheat the driver IC!
-const uint8_t RZ7886_DRAGBRAKE_DUTY = 100; // 0 - 100%. 100% = max. brake power while standing still. 100% is recommended for crawlers.
+uint16_t RZ7886_FREQUENCY = 500;     // 500 Hz is recommended. It is not audible, if virtual engine sound is running. Higher frequencies may overheat the driver IC!
```

Comment:
This variable is not configurable, right?
That's why the "const" is gone,
but can you also change the capitalization (lower case) if it's not longer a constant.

Actually, maybe you could do a:
#define RZ7886_FREQUENCY_INIT 500
and define the frequency variable in the .ino file.